app/views/teams/show.html.slim

```diff
@@ -86,16 +86,25 @@ nav.page
     h3 Supervisors
     p = @team.supervisors.any? ? link_to_team_members(@team, :supervisor) : content_tag(:em, 'Will be added by RGSoC Organizers')
 
-- if @team.conferences.in_current_season.any?
+- if @team.conference_preference.present?
   h4 Conference preferences
   ul.conferences
-  - @team.conference_preferences.sort_by(&:option).each do |preference|
-    li
-      = links_to_conferences([preference.conference]).first
-      = preference_option(preference)
+    - if @team.conference_preference.first_conference.present?
+      li
+        = links_to_conferences([@team.conference_preference.try(:first_conference)]).first
+        = content_tag :span, 'First conference', class: "label label-primary"
```

Comment:
I'd align this with the other view and speak of "First choice" and "Secondary choice" here.